docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,239 @@
+# Getting Started With Soci Snapshotter
+This document is meant to be a guide to walk through an example of using the
+soci-snapshotter.  To get everything running we will need to do a few things : 
+- start a local oras registry
+- push an image using ctr to our registry
+- create and push a soci index to our registry
+- run a lazily loaded container from the registry
+
+Along the way we will make sure to inspect each of our actions to ensure that
+they worked.  This is valuable to help learn about the tool and debugging.
+
+## Start a local ORA registry
+A standard OCI registry currently does not allow for arbitrary documents, and
+the soci indices and manifests are arbitrary.  The [ORAS
+project](https://oras.land/) tries to solve the project as in their words :
+
+> Registries are evolving as generic artifact stores. To enable this goal, the ORAS project provides a way to push and pull OCI Artifacts to and from OCI Registries.
+
+Given this we want to start our registry using an oci image from the oras
+project.  We will run it on port 5000 as an HTTP (**NOT** HTTPS) server without
+certificates.  Since this is just a locally run HTTP server we can interact with
+it using standard `curl` commands to confirm our actions.  
```

Comment:
I actually prefer having it higher up with the description.  The design of this document is to explain before invoking.

Reply:
Explaining before invoking makes good sense. But this paragraph doesn't scan the way it is written:

> Since this is just a locally run HTTP server we can interact with it using standard `curl` commands to confirm our actions. To do this we will use Docker :

In that last sentence, "this" is an anaphoric pronoun. To avoid ambiguity, it should be placed as near as possible to its antecedent. As written, "this" seems to be referring to the action of "interact[ing] with [the server], and then says that the tool for doing that is Docker. Obviously, that is not what you intended. But it is what you wrote.

Either reword the paragraph to resolve the ambiguity. Or split it into two paragraphs, since it contains two separate ideas: 1. starting the server; and 2) interacting with the server. Then, as you say, explain before invoking in both paragraphs.

Reply:
In the end I took your advice and split it